native/packages/react-native-bpk-component-button/src/BpkButton-test.android.js

```diff
@@ -60,6 +60,20 @@ jest.mock('./BpkButton', () => jest.requireActual('./BpkButton.android.js'));
 
 describe('Android', () => {
   commonTests();
+
+  it('secondary disabled buttons should have a transparent background', () => {
```

Comment:
this is common to both iOS and androind - consider renaming to "should render correctly with secondary type and disabled"